### PR TITLE
[release-2.5] Bump to Submariner 0.12.1

### PR DESCRIPTION
**Pull request by skitt**
Signed-off-by: Stephen Kitt <skitt@redhat.com>
(cherry picked from commit a0bf06af345bd29435475b79c0f647cd363c34aa)

### CODE REVIEW

File: pkg/hub/submarineragent/manifests/operator/submariner.io-submariners-cr.yaml [hunk 1]
```diff
@@ -42,3 +42,4 @@ spec:
 {{- end}}
   repository: registry.redhat.io/rhacm2
   version: v0.12.0
+  version: v0.12.1
```
Comment (sridhargaddam) [inline pkg/hub/submarineragent/manifests/operator/submariner.io-submariners-cr.yaml:45]: We are adding a second version. Is this valid or a typo?

Comment (skitt) [reply pkg/hub/submarineragent/manifests/operator/submariner.io-submariners-cr.yaml:44]: Oops, a mis-merge, thanks!

**Pull request merged**